06-func.md

```diff
@@ -22,10 +22,9 @@ and didn't want to generate a figure for every single one?
 Commenting out the figure-drawing code is a nuisance.
 Also, what if we want to use that code again,
 on a different dataset or at a different point in our program?
-Cutting and pasting it is going to make our code get very long and very repetative,
-very quickly.
-We'd like a way to package our code so that it is easier to reuse,
-and Python provides for this by letting us define things called 'functions' -
+Cutting and pasting it will make our code very long and very repetitive,
+very quickly whereas we would like a way to package our code that makes it easy to reuse.
+Python provides a solution for this by letting us define 'functions' -
```

Comment:
I'm +1 Azalee's rewording of this; putting the two sentences together makes it a bit long.